real-time-queue-observation-query/config.yml

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
A couple of quick thoughts.  What would a customer want to do with this example?  In your introduction, I would really like if you could provide some specific scenarios on why the code below would be useful.

I would also double-check grammar and spelling.  It looks like there were severe places that were missed during review.

Reply:
Thank you for the inputs. I have updated and review the content again. If you see any more changes that will help make this tutorial more informative. Let me know.